autoload/maktaba/plugin.vim

```diff
@@ -240,10 +242,10 @@ function! maktaba#plugin#Enter(file) abort
     return [l:plugin, 0]
   endif
 
-  if l:filedir !=# 'autoload'
+  if l:filedir !=# 'autoload' && has_key(l:plugin.flags, l:filedir)
```

Comment:
This change avoids triggering flags.vim to load where it's not needed and potentially wasteful/unsafe. Calling Flag could even try to load recursively if this maktaba#plugin#Enter was called from flags.vim.